docroot/modules/custom/foia_webform/tests/src/Kernel/FoiaSubmissionQueueHandlerTest.php

```diff
@@ -20,7 +20,7 @@
  *
  * @package Drupal\Tests\foia_webform\Kernel
  */
-class FoiaSubmissionServiceQueueHandlerTest extends KernelTestBase {
+class FoiaSubmissionQueueHandlerTest extends KernelTestBase {
```

Comment:
You changed the name of a class - are you sure this is correct?

Reply:
@brockfanning I changed the correct class name by looking into the suggestion